src/main/java/Sakura/Todo.java

```diff
@@ -0,0 +1,36 @@
+package Sakura;
+
+/**
+ * Represents a task to be done.
+ */
+public class Todo extends Task {
+
+    /**
+     * Constructor for the task.
+     *
+     * @param description description of the task to be completed.
+     */
+    public Todo(String description) {
+        super(description);
+    }
+
+    /**
+     * Convert the task into the specific string for data saving.
+     *
+     * @return the string representation in the specific format for database
+     */
+    @Override
+    public String stringifyTask() {

```

Comment:
```suggestion
    public String encodeTask() {
```

Hi Shawn, I think this renaming will help to communicate how `stringifyTask` is different from `toString`.

I liked the use of `String.format()`. It helped me immediately understand the encoding scheme. In comparison, the alternative would be `return ("T|" + this.isDone ? "1" : "0" + "|" + this.description)`, which is nowhere as elegant.